src/conn/conn_stat.c

```diff
@@ -221,6 +221,101 @@ err:	__stat_sources_free(session, &sources);
 }
 
 /*
+ * __statlog_print_header --
+ *	Write the header for statistics when running in JSON mode.
+ */
+static int
+__statlog_print_header(WT_SESSION_IMPL *session)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+
+	if (!FLD_ISSET(conn->stat_flags, WT_STAT_JSON))
+		return (0);
+
+	/*
+	 * This flag is required in order to generate correct JSON when printing
+	 * out stats for individual tables. When we are about to print the first
+	 * table's stats we must print out the wiredTigerTables header once
+	 * only and add a correct closing brace when we finish the tables
+	 * section. To do this we maintain a flag variable to note when we have
+	 * printed the first table. Unfortunately, the mechanism which we use
+	 * to print stats for each table does not allow passing of variables
+	 * by reference, this necessitates the use of a variable on the
+	 * connection. The variable is safe as the JSON printing logic is only
+	 * performed by the single threaded stat server.
+	 */
+	FLD_CLR(conn->stat_flags, WT_STAT_JSON_PRINT_TABLES);
+	WT_RET(__wt_fprintf(session, conn->stat_fs,
+	    "{\"version\":\"%s\",\"localTime\":\"%s\"",
+	    WIREDTIGER_VERSION_STRING, conn->stat_stamp));
+	return (0);
+}
+
+/*
+ * __statlog_print_table_name --
+ *	Write the header for the wiredTigerTables section of statistics if
+ *	running in JSON mode and the header has not been written this round,
+ *	then print the name of the table.
+ */
+static int
+__statlog_print_table_name(
+    WT_SESSION_IMPL *session, const char *name, bool conn_stats)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+
+	/*
+	 * If printing the connection stats, write that header and we are done.
+	 */
+	if (conn_stats) {
+		WT_RET(__wt_fprintf(
+		    session, conn->stat_fs, ",\"wiredTiger\":{"));
```

Comment:
I know that `wiredTiger` is camel case, but I prefer `WiredTiger` here and `WiredTigerTables` below. Remembering to start the storage engine with a lower case `w` always seems wrong.